lstn/static/js/controllers.js

```diff
@@ -383,6 +383,7 @@ angular.module('lstn.controllers', [])
       }
 
       // Play the track through Rdio
+      apiswf.rdio_setVolume(0.7);
```

Comment:
I've heard multiple reports that Lstn is stupid loud right now. I'm dropping it down to 70% for now as a work around until we can consider either adding UI for it or assuming that people have a number of ways to adjust the volume and let them control it out side of Lstn.